website/docs/r/file.html.md

```diff
@@ -10,12 +10,18 @@ description: |-
 
 Generates a local file with the given content.
 
-~> **Note** When working with local files, Terraform will detect the resource
+~> **Note about resource behaviour**
+When working with local files, Terraform will detect the resource
 as having been deleted each time a configuration is applied on a new machine
 where the file is not present and will generate a diff to re-create it. This
 may cause "noise" in diffs in environments where configurations are routinely
 applied by many different users or within automation systems.
 
+~> **Note about _sensitive_ content**
+We are deprecating the use of this resource for _sensitive_ content.
+If in need to use _sensitive_ content, please use the [`local_sensitive_file`](./sensitive_file.html)
+resource instead.
+
```

Comment:
We typically have just adjusted the actual attribute description for letting practitioners know about deprecations -- practitioners will automatically see the warnings with what to do when running Terraform. It looks like you already updated the attribute description, so I think this can just be informational.

```suggestion
-> If the file content is sensitive, use the [`local_sensitive_file`](./sensitive_file.html) resource instead.
```